synapse/config/jwt.py

```diff
@@ -16,13 +16,8 @@
 
 from synapse.types import JsonDict
 
-from ._base import Config, ConfigError
-
-MISSING_AUTHLIB = """Missing authlib library. This is required for jwt login.
-
-    Install by running:
-        pip install synapse[jwt]
-    """
+from ..util.check_dependencies import check_requirements
```

Comment:
```suggestion
from synapse.util.check_dependencies import check_requirements
```

When modules aren't in the current directory we usually prefer if the import path is absolute :)

This will need linting (because github's suggestions apparently don't allow me to move the import statement a line above :slightly_frowning_face: )